ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -269,6 +269,7 @@ public void changeActiveCard(final int cardPosition, final boolean animate) {
         changeActiveCard(getChildAt(mCardPositionOffset + cardPosition), animate);
     }
 
+    @UiThread
```

Comment:
The import for UiThread was missing.

Reply:
thanks, i cherry-picked the commit from somewhere it already existed. just got missed, but it would explain the travis failure :)